blaze/expr/collections.py

```diff
@@ -100,10 +100,32 @@ def sort(child, key=None, ascending=True):
     ascending : bool, optional
         Determines order of the sort
     """
+    if ascending not in (True, False):
```

Comment:
Ah, this is really helpful, I have tried to do that in the past. Can you just put a small comment in the source here that explains why we are doing this so I remember in the future?